src/test/error.ts

```diff
@@ -0,0 +1,32 @@
+import { expect } from "chai";
```

Comment:
convention in this repo is for the file to be named `src/test/path/file.spec.ts` (note the `spec`)

Reply:
Good catch, thanks. Fixed in 8e3886a91429a174f02d52bbc2d1efc80319ece6.